techlottus/portalverse-uane
# [DEV-5291] Migrate banner component

## Issue
 - Migrate banner component from UTEG to that repo

## Solution
 - [feat: Banner component migrated](https://github.com/techlottus/portalverse-uane/pull/149/commits/e392aa8705eaec305a422ff0a291e8343bb6a9a1)
 - [feat: oferta educativa banner image updated](https://github.com/techlottus/portalverse-uane/pull/149/commits/6ea9a3c1a5a65d7d180c0408337747548573ddb8)

## Testing
 - Go to `/nosotros/campus`
 - Change size of the browser and check that the banner image is not distorted
![image](https://user-images.githubusercontent.com/62124025/235014048-ed66b3de-c468-4e52-b4be-5881a7052b8e.png)
 - Keep rocking 🔥
